apps/synt-person-api/src/main/java/no/nav/registre/testnorge/synt/person/consumer/command/GetIdentCommand.java

```diff
@@ -0,0 +1,50 @@
+package no.nav.registre.testnorge.synt.person.consumer.command;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+import java.time.LocalDate;
+import java.util.concurrent.Callable;
+
+import no.nav.registre.testnorge.synt.person.consumer.command.model.CreateIdentRequest;
+
+@Slf4j
+@RequiredArgsConstructor
+public class GetIdentCommand implements Callable<String> {
+    private final WebClient webClient;
+    private final String applicationName;
+
+    @Override
+    public String call() {
+
+        log.info("Henter ident fra ident pool");
+
+        var request = CreateIdentRequest
+                .builder()
+                .antall(1)
+                .foedtEtter(LocalDate.now().minusYears(90))
```

Comment:
Magisk tall 90, hvorfor er det 90? Kunne man lagt inn denne som en variabel med navn for å forklare den bedre?